test/built-ins/String/prototype/Symbol.iterator/Symbol.toStringTag.js

```diff
@@ -0,0 +1,20 @@
+// Copyright (C) Copyright 2014 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 21.1.5.2.1
+description: >
+    The initial value of the @@toStringTag property is the string value "String
+    Iterator". This property has the attributes { [[Writable]]: false,
+    [[Enumerable]]: false, [[Configurable]]: true }.
+includes: [propertyHelper.js]
+---*/
+
+var StringIteratorProto = Object.getPrototypeOf(
+  String.prototype[Symbol.iterator]()
```

Comment:
`new String()[Symbol.iterator]()` or `""[Symbol.iterator]()` to avoid a `TypeError` with the current (rev37) semantics. (`String.prototype[Symbol.iterator]()` calls `String.prototype.toString()` which throws an error because `String.prototype` is not a string object.)